interface-definitions/service-ids-ddos-protection.xml.in

```diff
@@ -70,17 +70,34 @@
                   <multi/>
                 </properties>
               </leafNode>
-              <node name="mode">
+              <leafNode name="mode">
                 <properties>
                   <help>Traffic capture modes</help>
+                  <completionHelp>
+                    <list>mirror sflow</list>
+                  </completionHelp>
+                  <valueHelp>
+                    <format>mirror</format>
+                    <description>>Listen mirrored traffic mode</description>
```

Comment:
There's a stray closing `>` there. There's also missing preposition. I think "Listen to mirrored traffic" is a better wording.

Reply:
Fixed! thanks